README.md

```diff
@@ -106,6 +106,160 @@ of each other.
+
+### I got hits, why didn't I get an alert?
+
+If you got logs that had ``X query hits, 0 matches, 0 alerts sent``, it depends on the ``type`` why you didn't get any alerts. If ``type: any``, a match will occur for every hit. If you are using ``type: frequency``, ``num_events`` must occur within ``timeframe`` of each other for a match to occur. Different rules apply for different rule types.
+
+If you see ``X matches, 0 alerts sent``, this may occur for several reasons. If you set ``aggregation``, the alert will not be sent until after that time has elapsed. If you have gotten an alert for this same rule before, that rule may be silenced for a period of time. The default is one minute between alerts. If a rule is silenced, you will see ``Ignoring match for silenced rule`` in the logs.
+
+If you see ``X alerts sent`` but didn't get any alert, it's probably related to the alert configuration. If you are using the ``--debug`` flag, you will not receive any alerts. Instead, the alert text will be written to the console. Use ``--verbose`` to achieve the same affects without preventing alerts. If you are using email alert, make sure you have it configured for an SMTP server. By default, it will connect to localhost on port 25. It will also use the word "elastalert" as the from address. Some SMTP servers will rejects this because it does not have a domain while others will add their own domain automatically. See the email section in the documentation for how to configure this.
+
+### Why did I only get one alert when I expected to get several?
+
+There is a setting called ``realert`` which is the minimum time between two alerts for the same rule. Any alert that occurs within this time will simply be dropped. The default value for this is one minute. If you want to receive an alert for every single match, even if they occur right after each other, use
+
+```
+realert:
+  minutes: 0
+```
+
+You can of course set it higher as well.
+
+### How can I prevent duplicate alerts?
+
+By setting ``realert``, you will prevent the same rule from alerting twice in an amount of time.
+
+```
+realert:
+  days: 1
+```
+
+You can also prevent duplicates based on a certain field by using ``query_key``. For example, to prevent multiple alerts for the same user, you might use
+
+```
+realert:
+  hours: 8
+query_key: user
+```
+
+Note that this will also affect how many rule types work. If you are using ``type: frequency`` for example, ``num_events`` for a single value of ``query_key`` must occur before an alert will be sent. You can also use a compound of multiple fields for this key. For example, if you only wanted to receieve an alert once for a specific error and hostname, you could use
```

Comment:
"how many rule types work" is a bit misleading, like meaning some types might stop working. Change it to something like "the way many rule types work".